spec/support/request_spec_helper.rb

```diff
@@ -12,6 +12,20 @@ def version(ver = 'v1.0')
   end
 
   DEFAULT_USER = {
+    "entitlements" => {
```

Comment:
This `DEFAULT_USER` constant shouldn't be needed if 
```
Dir[ManageIQ::API::Common::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
```
Is included. The `default_user_hash` methods in `UserHeaderSpecHelper` will cover all the behavior around the default_user_hash

Reply:
Yeah I tried to include the UserHeaderSpecHelper just like we do on catalog, but it isn't acting properly so I fell back to just updating the other hash for now.